app/views/shared/_eligibility_checker.erb

```diff
@@ -0,0 +1,9 @@
+<br>
+<h1 class="govuk-heading-m">Eligibility checker</h1>
+<p class="govuk-hint">Which route will applicants take through the eligibility checker?</p>
+<%= f.govuk_radio_button :eligibility_enabled, 'yes', label: { text: 'Standard – Asks 5 standard questions that most countries would need to be asked.' }, link_errors: true %>
+<%= f.govuk_radio_button :eligibility_enabled, 'no', label: { text: 'Reduced – Only asks if the applicant’s teaching qualification was completed in their country of recognition.' } %>
```

Comment:
I don't think this field maps to `eligibility_enabled` as that dictates whether the user sees an eligible or ineligible page. Instead this relates to `eligibility_skip_questions`, which if `true` means that it's the reduced flow.